.github/workflows/nix-update-flake.yml

```diff
@@ -0,0 +1,22 @@
+name: Update nix flake.lock
+on:
+  workflow_dispatch: # allows manual triggering
+  schedule:
+    - cron: '0 0 * * 0' # runs weekly on Sunday at 00:00
```

Comment:
Can you pick a random time instead, either during the 00 hour or during some time period? There are probably way too many things happening at that moment in time.

Normal `cron` supports `?` instad of `0` for this, I think; I'm not sure if Github's does, but it's likely. Or, you could pick a random number between 0 and 60 as in https://xkcd.com/221/.

Reply:
Frankly I'd expect GitHub to load shed this appropriately themselves (e.g. each cron tab entry would get spread out over a normal +/- 5min distribution or something like that.) Do we really need something like this? I wouldn't be surprised if there was some precedent for this behavior but I'm honestly not sure if it's something we should care about.

Reply:
I don't know for sure. If you have reason to believe it's okay, it might be fine.

Reply:
Unfortunately, I don't think GitHub does support those operators: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
I went ahead and just changed it to 3:40 UTC, which is totally arbitrary, but should be fine.